public/app/features/alerting/unified/components/receivers/grafanaAppReceivers/types.ts

```diff
@@ -12,4 +12,5 @@ export const GRAFANA_APP_RECEIVERS_SOURCE_IMAGE: Record<SupportedPlugin, string>
   [SupportedPlugin.Incident]: '',
   [SupportedPlugin.MachineLearning]: '',
   [SupportedPlugin.Labels]: '',
+  [SupportedPlugin.Slo]: '',
```

Comment:
We don't need this line on this PR. Slo is not going to become a Grafana receiver.

Reply:
I see! Initially, I did not add this line, but without it, drone fails. Any recommendation?